src/libstd/time/duration.rs

```diff
@@ -188,6 +188,13 @@ impl DivAssign<u32> for Duration {
     }
 }
 
+#[stable(feature = "duration_default", since = "1.9.0")]
+impl Default for Duration {
+    fn default() -> Duration {
+        Duration::new(0, 0)
```

Comment:
My [question on StackOverflow](https://stackoverflow.com/questions/36477505/why-is-default-not-implemented-for-mutex-rwlock-condvar-duration) has brought up the concern that a 0-duration is *not* a sensible default in most cases. Using this e.g. as a timeout is bound to fail at runtime. It's probably better not to implement Default for Duration at all.